stable/kube-state-metrics/templates/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```

Comment:
We've started transitioning the `app` label to `{{ template "name" . }}` for better label querying. Let's do that here.